src/lib/user_command_input/user_command_input.mli

```diff
@@ -74,6 +74,7 @@ val to_user_command :
   -> get_account:(Account_id.t -> Account.t option Participating_state.T.t)
   -> constraint_constants:Genesis_constants.Constraint_constants.t
   -> logger:Logger.t
+  -> signature_kind:Mina_signature_kind.t
```

Comment:
This probably better off be part of the `User_command_input.t`, I would argue. However that would break the daemon RPC, and would be a HF. 

If we want code to be clean I can do that instead.